### DISCUSSION TITLE
Changing the Author Code via API with Only Xbox Account Access

### DISCUSSION THREAD

**Discussion by rodor03** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#288] [Q&A]
Hi, I wanted to ask if it is possible to change the author code via API, having access only to the xbox account?

**Comment by LeleDerGrasshalmi** [comment]
Hi, that what do you mean by "author code"? Also that sounds a bit sketchy, we do not tolerate account sellers here are are not willing to help them.

**Comment by rodor03** [comment]
I meant the "creator code" that is used in Fortnite to support a specific content creator through the Support-A-Creator program. I apologize for any confusion. My question is about the API usage to change or retrieve this code programmatically, not related to account selling or anything against the community guidelines.
